CI.hs

```diff
@@ -0,0 +1,88 @@
+-- XXX IDEALLY:
+-- Right now this is very minimal and has no external dependencies.
+-- Use streamly
+-- Move this into benchmark/
+-- Add an executable to streamly-bencharks or merge this with Chart.hs
```

Comment:
Chart.hs should go away once bench-show is made generic enough to handle the cases in it.

Reply:
This should not be reviewed.